frontend/src/pages/home/resources/useResourcesSection.tsx

```diff
@@ -70,6 +70,10 @@ export const useResourcesSection = (): React.ReactNode => {
                   key={`${doc.metadata.name}`}
                   odhDoc={doc as unknown as OdhDocument}
                   showFavorite={false}
+                  style={{
+                    border: '1px solid var(--pf-v5-global--BorderColor--100)',
+                    borderRadius: 16,
+                  }}
```

Comment:
Hmmm... This feels like a sneaky addition -- can you explain the reason for this change?

Reply:
Looks like based on UX's feedback (missed those comments lol) -- this is a fine change -- next time we shouldn't be addressing random unrelated feedback in the PRs as it pollutes the tickets & PRs and makes it hard to trace origins.